static/sass/_snapcraft_details_screenshots.scss

```diff
@@ -0,0 +1,103 @@
+@mixin snapcraft-details-screenshots {
+  .p-screenshots {
+    position: relative;
+
+    .p-screenshots__wrapper {
+      // sass-lint:disable-block no-vendor-prefixes
+      -ms-overflow-style: -ms-autohiding-scrollbar;
+      -webkit-overflow-scrolling: touch; // sass-lint:disable-line no-misspelled-properties
+
+      display: flex;
+      flex-wrap: nowrap;
+      overflow-x: auto;
+      overflow-y: hidden;
+    }
+
+    .p-screenshot {
+      flex: 0 0 auto;
+      height: 240px;
```

Comment:
`max-height` maybe?

Reply:
Tried that, but it causes some issues.
First of all layout 'jumps' before images load, because they don't have any height until they load.
And there are some flexbox issues in Firefox. It seems like it messes up the size of this element when it doesn't have a fixed height given in CSS.

Reply:
Hmm, that's annoying - can we init the screenshots UI after the final image has loaded? That way it should all be based on the final sizes (unless you resize the browser like a madm'n)

Reply:
We can, but what would be the benefit of doing that?

If we don't set the size of screenshots in CSS it will render as nothing until images start popping up which will make the whole page jump. Waiting for last image to load would not change that.

These are screenshots, apart from some weird cases they will be above 240px, so what's the point of using `max-height`?